server/controllers/ssotRetrievalController.js

```diff
@@ -5,52 +5,58 @@ const userAccessModels = require('../models/userAccessObjectModel.js');
 
 // GET /ssot/:id
 exports.getSingleSourceOfTruth = async (req, res) => {
-    try {
-        res.set('Content-Type', 'application/json');
-        
-        const { id } = req.params;
-        const ssot = await mongoose.model('SSoT').findById(id).exec();
-        res.send(ssot);
-    } catch (err) {
-        console.error(err);
-    }
+  try {
+    res.set('Content-Type', 'application/json');
+
+    const { id } = req.params;
+    const ssot = await mongoose.model('SSoT').findById(id).exec();
+    res.send(ssot);
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
As we all start to use prettier now, should we maybe agree on an indentation amount? So 4 vs 2 spaces? Otherwise, these changes will make Pull Requests unnecessarily long.